core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -187,11 +187,15 @@ class PlaintextConsumerTest extends BaseConsumerTest {
     assertEquals(1, listener.callsToAssigned)
     assertEquals(0, listener.callsToRevoked)
 
-    Thread.sleep(3500)
+    // as long as we are part of max.poll no rebalance should be triggered
+    Thread.sleep(500)
```

Comment:
Can we replace this with `waitUntilTrue` as well?

Reply:
The num.assignment is already 1 here, and the first sleep is to verify that it did not change; after second thought I think I'll just remove it since without polling there would be no rebalance anyways.